Packs/Elasticsearch/Integrations/Elasticsearch_v2/README.md

```diff
@@ -244,12 +248,12 @@ Returns the health status of the integration. This commmand should be used for d
 
 #### Input
 
-There are no input arguments for this command.
+| **Argument Name** | **Description** | **Required** |
```

Comment:
why include an empty table? Recommend just saying (no bullet)
There are no input arguments for this command.